cms/djangoapps/contentstore/views/course.py

```diff
@@ -1521,15 +1521,32 @@ def group_configurations_list_handler(request, course_key_string):
             else:
                 experiment_group_configurations = None
 
-            content_group_configuration = GroupConfiguration.get_or_create_content_group(store, course)
+            all_configurations = GroupConfiguration.get_all_content_groups(store, course)
+            should_show_enrollment_track = False
+            group_schemes = []
+            for config in all_configurations:
+                group_schemes.append(config['scheme'])
+                if config['scheme'] == ENROLLMENT_SCHEME:
+                    enrollment_track_configuration = config
+                    should_show_enrollment_track = len(enrollment_track_configuration['groups']) > 1
+
+                    # Remove the enrollment track config and add it to the front of the list if it should be shown.
+                    all_configurations.remove(config)
+                    if should_show_enrollment_track:
+                        all_configurations.insert(0, config)
+
+            # Add empty content group if list returned is empty.  This will add ability to add new groups in the view.
+            if len(all_configurations) == 0 or COHORT_SCHEME not in group_schemes:
```

Comment:
This can be simplified to just COHORT_SCHEME not in group_schemes. If there is no Content Group Configuration, we want to create an empty one.